src/views/Home/Home.js

```diff
@@ -8,11 +9,28 @@ import './Home.css'
 export default function Home() {
   // add useState calls here for title, subtitle, font, align, and text
 
+  const [title, setTitle] = useState('')
+  const [font, setFont] = useState('')
+  const [subtitle, setSubtitle] = useState('')
+  const [align, setAlign] = useState('')
+  const [text, setText] = useState('')
+
```

Comment:
Great job on this one! Don't forget that you can set default values in your useState() calls. This comes in especially handy for the font and align properties.